src/pages/all.jsx

```diff
@@ -39,8 +39,8 @@ const AllPage = () => {
                     <div className="col">
                         <h2>Tell Your Story</h2>
                         <p>&quot;Jenkins Is The Way&quot; is a global showcase of how developers and engineers are building, deploying, and automating great stuff with Jenkins.</p>
-                        <p>Share your story and we’ll send you a free Jenkins Is The Way T-Shirt.</p>
-                        <p>Our short form will capture details about your project’s goals and technical challenges, and the unique solutions you came up with using Jenkins.</p>
+                        <p>Share your story and we'll send you a free Jenkins Is The Way T-Shirt.</p>
```

Comment:
```suggestion
                        <p>Share your story and we'll send you a free Jenkins Is The Way t-shirt.</p>
```
updating casing for rebuild